README.md

```diff
@@ -442,5 +443,8 @@ Distributed under the Apache License, Version 2.0. See [LICENSE](LICENSE) for mo
 
 [_video_demo_filetable_url]: https://groups.google.com/g/dataverse-community/c/w_rEMddESYc/m/6F7QC1p-AgAJ
 
+<!-- September, 2024 -->
+[_video_demo_collectionpage_creationpages_fileuploading_url]: https://groups.google.com/g/dataverse-community/c/heSnBRhhriM/m/bFgLDvvZAQAJ
```

Comment:
I don't understand why these links are repeated here. Should we just delete them? Or add a comment about why they need to be maintained in two places?

Reply:
Perfect, thanks.